pages/about/jobs.md

```diff
@@ -11,7 +11,19 @@ eleventyNavigation:
   title: .Gov jobs
 ---
   
-We are a team of program managers, technologists, and analysts. We help U.S. government organizations gain public trust by making .gov a well-known, reliable, and secure space online.
+The .gov program at the Cybersecurity and Infrastructure Security Agency (CISA) is a team of program managers, technologists, and analysts. We help U.S. government organizations gain public trust by making .gov a well-known, reliable, and secure space online.
+
+The .gov top-level domain (TLD) is only available to U.S.-based government organizations and publicly controlled entities. The .gov team has a direct impact on the security of thousands of online services relied on by federal, state, tribal, territorial, and local governments throughout the United States – and their millions of users. 
+
+We help shape how governments manage and secure their DNS infrastructure, how the .gov registry authenticates and authorizes registrant organizations and domains, and how CISA maintains insight into a trusted and trustworthy namespace for governments.
+
+Learn [how to apply for jobs at CISA and get tips for preparing your resume](https://www.cisa.gov/careers/resume-application-tips).
+
+## Open positions
+- Position 1
```

Comment:
**comment (non-blocking)**: I think I would rather this section be empty than having these placeholders that aren't actual position names.